runtime/src/iree/hal/drivers/hip/event_semaphore.c

```diff
@@ -295,6 +295,10 @@ iree_status_t iree_hal_hip_semaphore_multi_wait(
       // it is because the event has already been signaled to that value.
       if (!cpu_events[i]) {
         semaphore_hit = true;
+        if (iree_hal_hip_semaphore_is_aborted(semaphore_list.semaphores[i])) {
+          status = iree_make_status(IREE_STATUS_ABORTED,
```

Comment:
Fyi if something will happen outside of error cases (I don't think this will) you'll want to use iree_status_from_code as iree_make_status is expensive (may take a full stack trace/etc).